### DISCUSSION TITLE
feed/article/QYnuqHAzewVwqEWYNhkQ

### DISCUSSION THREAD

**Discussion by giscus** [wisdomekpotu/devhouse#13] [Comments]
# feed/article/QYnuqHAzewVwqEWYNhkQ

Web site created using create-react-app

http://localhost:3000/feed/article/QYnuqHAzewVwqEWYNhkQ

**Comment by wisdomekpotu** [comment]
nice article

**Reply by wisdomekpotu** [reply]
i want to suggest some new features
